Documentation/compatibility/wpf-tabcontrol-selectionchanged-and-selectedcontent.md

```diff
@@ -0,0 +1,51 @@
+## TabControl SelectionChanged event and SelectedContent property
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+When its selection changes, a `T:System.Windows.Controls.TabControl` now updates the value of its
```

Comment:
When its selection changes --> Starting with the .NET Framework 4.7.1, when its selection changes
"now" can be removed